src/components/GlMap.svelte

```diff
@@ -27,6 +27,11 @@
       await import('mapbox-gl/dist/mapbox-gl.css');
       renderer = await import('mapbox-gl');
       renderer.accessToken = $configStore.mapboxGlAccessToken;
+
```

Comment:
@aparlato @ebrelsford — My QA samples do in fact set `config.API_URL`, but the original Stamen examples do not.  My goal here was to try/catch/swallow if the variable is not set.  In Stamen case, this code will error, then do nothing.  In my sample, with a custom Atlas server, it will set the URL work famously.  See usage for my [atlas config](https://github.com/roblabs/maperture/blob/gh-pages/config/local.atlas.js).